rskj-core/src/main/java/co/rsk/peg/federation/FederationStorageProviderImpl.java

```diff
@@ -385,6 +398,18 @@ private void saveProposedFederation(ActivationConfig.ForBlock activations) {
         bridgeStorageAccessor.saveToRepository(PROPOSED_FEDERATION.getKey(), proposedFederation, BridgeSerializationUtils::serializeFederation);
     }
 
+    private void saveFundTransactionUnsignedHash(ActivationConfig.ForBlock activations) {
+
+        if (!activations.isActive(RSKIP419) || !isFundTransactionUnsignedHashSet) {
+            return;
+        }
+
+        byte[] data = nonNull(fundTransactionUnsignedHash) ? BridgeSerializationUtils.serializeSha256Hash(fundTransactionUnsignedHash) : null;
```

Comment:
```suggestion
        byte[] data = Optional.ofNullable(fundTransactionUnsignedHash)
                      .map(BridgeSerializationUtils::serializeSha256Hash)
                      .orElse(null);
```
I'm liking this approach of using Optional to handle null checks, looks cleaner to me at least. What do you think?

Reply:
I agree.